build.xml

```diff
@@ -46,4 +46,13 @@
     <mkdir dir="${build.conf.dir}"/>
     <copy file="${tz.data}" todir="${build.conf.dir}"/>
   </target>
+
+  <target name="pkg" depends="timezones">
+    <delete dir="build/dist"/>
+    <delete dir="build/stage"/>
+    <exec dir="." executable="./pkg-builder.pl" failonerror="true">
+       <arg value="--define"/>
+       <arg value="jetty.distro=jetty-distribution-9.3.5.v20151012"/>
```

Comment:
not sure but just a thought
are we making upgrading jetty a little bit harder by putting version numbers everywhere? can we use some global variable defined by build.pl?